.github/workflows/js.yaml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+      - '**'
+      - '!master'
+
+name: wasmerjs
```

Comment:
> I'm saying that to avoid confusion with wasmer-js, https://github.com/wasmerio/wasmer-js/.

This is indended to replace `wasmer-js`. That's why I chose this name :)

Reply:
This is the job name, no need to change it as I'm not sure Github allows `-` inside job names